federation/pkg/kubefed/init/init_test.go

```diff
@@ -690,19 +692,33 @@ func fakeInitHostFactory(federationName, namespaceName, ip, dnsZoneName, image,
 								},
 							},
 						},
-						{
-							Name: "etcddata",
-							VolumeSource: v1.VolumeSource{
-								PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
-									ClaimName: pvcName,
-								},
-							},
-						},
 					},
 				},
 			},
 		},
 	}
+	if etcdPersistence == "true" {
```

Comment:
I think having to duplicate code in a test is a sign of either poorly-factored code or a bad test.  Will follow up separately.